ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -396,21 +396,21 @@
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseEventID": "amendRespondentsGatekeeping",
     "UserRole": "caseworker-publiclaw-gatekeeper",
-    "CRUD": "CRU"
+    "CRUD": "R"
```

Comment:
what about having new file for old events which are maintained only for backward compatibility? maybe
AuthorisationCaseEvent-deprecated.json

Reply:
updated with `Comment`. A separate file would have been preferred but this won't work due to duplicate `CaseEventID`